docs/reference/migration/index.asciidoc

```diff
@@ -3,23 +3,11 @@
 
 [partintro]
 --
-This section discusses the changes that you need to be aware of when migrating
-your application from one version of Elasticsearch to another.
+This section discusses the changes that you need to be aware of to migrate
+your application to {version}. For more information about what's new in this
+release, see the <<release-highlights>> and <<es-release-notes>>.
 
```

Comment:
It might also be nice to have a bulleted list on this page, especially as the number of releases grows.  Similar to https://www.elastic.co/guide/en/kibana/7.0/breaking-changes.html